env0/resource_template_test.go

```diff
@@ -60,6 +60,7 @@ func TestUnitTemplateResource(t *testing.T) {
 		TerragruntVersion:  "0.35.1",
 		IsGitlabEnterprise: true,
 		TerraformVersion:   "0.15.1",
+		IsTerragruntRunAll: true,
```

Comment:
Updated the acceptance tests to check IsTerragruntRunAll.

Reply:
so now we don't have a non-run-all check? shouldn't it be added on top of a low version of terragrunt that should run as usual, a higher version (>=0.28.1) that isn't run-all that should work, the added low version that is run-all and should fail? total 4 cases (low-all, high-all, low-not-all, high-not-all). that seems to replace one case with another, without adding the counterpart.

Reply:
@sagilaufer1992 - by default IsTerragruntRunAll is false. Therefore, there are some testcases that run with that mode.
I can change one of them to a lower version terragrunt. That should cover all 4 use-cases.

Reply:
as we check versions as well, and testing shouldn't take long (unit testing, right?) I'd rather we'll check all 4 use-cases.

Reply:
8c55b175bd3f06253fd8909710e9d8ee39dca700

Reply:
All 4 use-cases are tested.

gleeUpdatedTemplate - high-all
gitlabUpdatedTemplate - low-not-all
bitbucketUpdatedTemplate - high-not-all
"run all with outdated terragrunt version" - low-all (which is expected to fail).

Reply:
@sagilaufer1992

Reply:
was about to comment that I found all, and thank you. so extra thank you for making it easier for others as well 🙏